src/polycubed/src/cube_factory_impl.cpp

```diff
@@ -193,5 +193,13 @@ std::vector<std::shared_ptr<BaseCubeIface>> CubeFactoryImpl::get_cubes() {
   return r;
 }
 
+std::vector<std::string> CubeFactoryImpl::get_names_cubes() const {
```

Comment:
I know that there are not many comments in Polycube. However, it would be nice to add something more in order to improve the code.
So, for instance, what does this function?

Reply:
Added :+1: